src/libraries/System.Security.Cryptography.Xml/src/System/Security/Cryptography/Xml/KeyInfoX509Data.cs

```diff
@@ -30,7 +30,15 @@ public class KeyInfoX509Data : KeyInfoClause
 
         public KeyInfoX509Data(byte[] rgbCert)
         {
-            X509Certificate2 certificate = new X509Certificate2(rgbCert);
+            // Compat: this accepts null arrays for certificate data and would not throw. X509CertificateLoader throws
```

Comment:
This is a little odd, but we have a unit test for this asserting that a null `rgbCert` does not throw. Since it's S.S.C.Xml I thought the best thing to to err on preserving existing behavior as much as possible.